document/CoreFormat.mdk

```diff
@@ -348,38 +352,38 @@ This shall be one of the following:
 *	`Tiger`
 *	`Whirlpool`
 
-## `issue` object { #issue }
+## `result` object { #result }
 
-### General { #issue-General }
+### General { #result-General }
 
-An `issue` object describes a single issue detected by a static analysis tool.
+A `result` object describes a single result produced by a static analysis tool.
 
-### `ruleId` property { #issue-ruleId }
+### `ruleId` property { #result-ruleId }
 
-An `issue` object shall contain a property named `ruleId` whose value is a string containing
-a stable, opaque identifier for the rule that was violated.
+A `result` object shall contain a property named `ruleId` whose value is a string containing
+a stable, opaque identifier for the rule that was evaluated to produce the result.
 
 ~ IsoExample
 `"CA2101"`
 ~
 
-### `fullMessage` property { #issue-fullMessage }
+### `fullMessage` property { #result-fullMessage }
 
-An `issue` object shall contain a property named `fullMessage` whose value is a string that describes the issue.
+A `result` object shall contain a property named `fullMessage` whose value is a string that describes the result.
 
-The `fullMessage` property should provide sufficient details to allow an end user to resolve the issue.
-In particular, `fullMessage` shall include all of the following information that is available and relevant to the issue:
+The `fullMessage` property should provide sufficient details to allow an end user to resolve any problem that the result might indicate.
+In particular, `fullMessage` shall include all of the following information that is available and relevant to the result:
 
-*	Information sufficient to identify the analysis target, and the location within the target where the issue occurred.
+*	Information sufficient to identify the analysis target, and the location within the target where the result occurred.
```

Comment:
since we have defined this section to refer to results that actually indicate an issue, let's fallback to 'problem' in the explanatory text